### DISCUSSION TITLE
How to add a new settings to Lively

### DISCUSSION THREAD

**Discussion by ayabusa** [rocksdanister/lively#1802] [General]
Hello,
I would like to add a new settings to Lively and don't really know how. By looking at the code I saw that it might be in Lively.Models in the file SettingsModel.cs. But is it the only place where I need to add my settings ?

**Comment by rocksdanister** [comment]
Are you trying to submit the changes to the main project or is this a personal fork?

**Comment by ayabusa** [comment]
For now it's a personal project but once I finish it, I might create a pull request of it.

**Comment by rocksdanister** [comment]
SettingsModel, GRPC server, client and proto files will need updating.

This should help:
https://github.com/rocksdanister/lively/wiki/Building

**Reply by ayabusa** [reply]
Thank you, I'm going to take a look at it !
